source/simulator/solver.cc

```diff
@@ -815,6 +815,7 @@ namespace aspect
             // processors
             catch (const std::exception &exc)
               {
+                computing_timer.exit_section();
```

Comment:
please move the statement to line 843, right before the `// avoid a deadlock` comment. This way, it covers all of the code here as well.